### ISSUE TITLE
User should not be able to change rastersource access_modifier after creation

### ISSUE THREAD

**Issue by Derryrover** [nens/lizard-management-client#603]
User should not be able to change raster-source acces-modifier after creation.

Right now on staging the user is able to do this.

Todo:
- verify that this is indeed a requirement. 
- disable this change on the client
- make this change on the backend unauthorised ?

**Comment by Derryrover** [comment]
@byrman 
@LirryPinter 
@joeriverheijden 
@lexvand 

Hi,
Made this ticket based on last meeting.
Do we know for sure we want the accessebilty (acces_modifier) field to be read-only after creation?
Do we also need to prevent changing acces_modifier on the backend via the api or is client side disabling it enough?

For now I assume that just client side disabling it is enough, but please let me know if you think otherwise.

kr Tom

**Comment by hoanphungt** [comment]
@lexvand 
Hi Lex, is it a requirement that user is not allowed to change the "Access Modifier" of an existing raster source? If this is the case, I can disable this functionality in the client. If we also want to unauthorise this change from the backend, then I can create a new ticket for backend.

**Comment by lexvand** [comment]
@hoanphungt Best to only prevent this in the frontend right now. When we start working on Geoblocks management we may reconsider this.
